.github/ISSUE_TEMPLATE/e2e_test_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: e2e test report
+about: Report an e2e test failure.
+title: ''
+labels: ['category: e2e', 'needs triage']
+assignees: ''
+
+---
+
+### Description
+<!-- Before reporting an issue, are you sure this e2e test is failing consistently? -->
+<!-- Add a clear and concise description of what the issue is. Please, be as descriptive as possible. -->
+
+**Error**:
+<!-- The error that the e2e test indicates. -->
+
+**Spec**:
+<!-- The test file that's triggering this error. -->
+
+**Relevant line(s) of code**:
+<!-- Line of code that is triggering this error. -->
+
+**GitHub jobs where this spec is failing**:
+<!-- Can you list all the jobs where the test failing? e.g. "WC - latest | wcpay - shopper", "WC - beta | wcpay - shopper" etc. -->
+- 
+
+### Additional context
+<!-- Any additional context or details you think might be helpful. -->
+<!-- Ticket numbers/links, plugin versions, system statuses etc. -->
+
+---
+
+- [ ] Added priority label.
+<!-- For the priority label, use your best judgement. -->
+- [ ] Added e2e label.
+<!-- Add one of the "e2e: broken flow/test/environment" labels. See their description and try to match it with the actual issue. -->
```

Comment:
I don't think we should expect this – it makes this a ~30 min task rather than a ~10 min task. I think it should be optional, and required when the issue is closed. 

I understand our concern here is that we want accurate data. Worst case: if it's missing (😮), it should be simple for e.g. the person who wants accurate data to add it later, by reading the fix PR or possibly chatting to someone (i.e. if their fix PR was not documented clearly). TLDR an audit process (later) would work better here than more expectations at the start (on reporter).

Let's outline the whole process required of a broken e2e reporter, and be realistic about how that could disrupt their flow. My dream is that it's zero friction, in reality we have a few tasks, each with multiple steps:

1. Investigate to work out which test(s) need skipping.
2. Push a PR to skip the broken test.
3. Open an issue to log that the test is broken and needs follow up (i.e. use this template).

My expectation is that this would take 5-10 min in the best possible case, and 15-30 would be a good allowance. Requiring an accurate or even best-guess `e2e-fail` label adds overhead.

Side note – maybe we should use `e2e-fail` or `e2e-broken` prefix for these new labels.

Reply:
I still think it's good to have an initial input by the logger, or we may end up in the worst case, where the label is missing.

>it should be simple for e.g. the person who wants accurate data to add it later, by reading the fix PR or possibly chatting to someone

I don't think this is an easy undertake, especially for someone who doesn't have context on the issue nor on the fix. Imagine a scenario where this person have dozens of issues like this and none of them have this label? They'd take a couple of days to gather all that information.

I sort of mixed your previous comment idea with this one and came into a middle-ground option. That would be asking the logger to label it using a quick judgement, so we don't take much of their time. At the same time, we leave an explicit ask for the assignee, who will have most context and more accurate information, to review this input.

If the assignee mark the checkbox, no further action is needed and we have accurate data. Otherwise, everyone's aware that this label may not be accurate, and anyone can ping the assignee later.

I applied this change in https://github.com/Automattic/woocommerce-payments/pull/8420/commits/aa8441ac331c55109d9354eaa70708f05b510598 and it will render like this:

<img width="646" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/23d46eb7-b319-4a80-bcc8-16a826c287aa">

Reply:
Sounds good. Adding the label should get easier as it gets more close to fixed.

Reply:
That checkbox wording is a little strange - because it assumes the label is usually correct. I would just ask what we want – say something like:

- Please ensure when closing this issue (PR fix) that only one `e2e-fail` label is added and it is accurate. 

Keep it simple.

There's no bulletproof way to get these labels right with wording or process, ultimately we have to ask and trust (and be prepared to fix errors).

Reply:
I adjusted it to your suggestion in https://github.com/Automattic/woocommerce-payments/pull/8420/commits/e542a5cc2ca7e2e779f8831e93454359b3e66318. I agree there's no bulletproof method, but I left the checkbox because I think it will help the assignee to visualize that todo and, when analyzing data, we can be sure the label has been reviewed.